sdk/core/azure-core/tests/async_tests/test_streaming_async.py

```diff
@@ -180,7 +180,26 @@ async def test_decompress_compressed_header(http_request):
         decoded = content.decode("utf-8")
         assert decoded == "test"
 
-
+@pytest.mark.asyncio
+@pytest.mark.parametrize("http_request", HTTP_REQUESTS)
+async def test_compress_compressed_no_header_offline(port, http_request):
+    # expect compressed text
+    client = AsyncPipelineClient("")
+    async with client:
+        request = http_request(method="GET", url="http://localhost:{}/streams/compressed_no_header".format(port))
+        pipeline_response = await client._pipeline.run(request,stream=True)
+        response = pipeline_response.http_response
+        data = response.stream_download(client._pipeline,decompress=False)
```

Comment:
so to clarify this test, we expect the server to return a body of compressed text. We specifically say "do not decompress", then we want to make sure it fails on decode, right? This makes sense to me, just confirming the intent of the test